src/lz4frame.jl

```diff
@@ -94,8 +105,8 @@ struct LZ4F_preferences_t
     reserved::NTuple{4, Cuint}        
 end
 
-function LZ4F_preferences_t(frame_info::LZ4F_frameInfo_t; compressionlevel::Cint=(Cint)(0), autoflush::Cuint=(Cuint)(0))
-    LZ4F_preferences_t(frame_info, compressionlevel, autoflush, (0,0,0,0))
+function LZ4F_preferences_t(frame_info::LZ4F_frameInfo_t; compressionlevel::Int=0, autoflush::Bool=false)
```

Comment:
Should be `Integer` not `Int`. If these are suppose to be mirroring C code you may want to leave these using `Cuint` and do the conversion at a higher level.